VRCFaceTracking/Params/Data/UnifiedData.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Text.Json.Serialization;
+using System.Windows;
+
+namespace VRCFaceTracking.Params
+{
+    /// <summary>
+    /// Struct that represents a single eye.
+    /// </summary>
+    public struct UnifiedSingleEyeData
+    {
+        // Eye data.
+        public Vector2 Gaze;
+        public float PupilDiameter_MM;
```

Comment:
Needs to be in CM and not MM (to make it clear for module devs and intent of use), there isn't any intrinsic logic to translate this to a 0-1 range. Making this in CM would mean that pupil dilation would definitely be between 0-1.

Reply:
Actually the diameter is exposed via an explicit implementation (like with eye gaze and the others), it really could just be altered in the merger class.